src/commands/domain/get.js

```diff
@@ -0,0 +1,67 @@
+const { flags } = require('@oclif/command')
+const { getDomainIdentifierArg, reqType, resolvedParam, splitPathParams } = require('../../support/command/parse-input')
+const { from, hasJsonStructure, prettyPrintJSON } = require('../../utils/general')
+const { getDomain } = require('../../requests/domain')
+const { getResponseContent } = require('../../support/command/handle-response')
+const BaseCommand = require('../../support/command/base-command')
+
+class GetDomainCommand extends BaseCommand {
+  constructor(...props) {
+    super(...props)
+    this.logDefinition = this.logDefinition.bind(this)
```

Comment:
Binding methods to the class context in the constructor is not a good pattern. Use arrow function instead.

`logDefinition(response) {` -> `logDefinition = (response) => {`

Reply:
PR updated.

Reply:
I mean just remove constructor and replace line 14 with mentioned code.
So it will becom 
```
class GetDomainCommand extends BaseCommand {
  logDefinition = (response) => {
    const definition = getResponseContent(response)

    this.log(hasJsonStructure(definition)
      ? prettyPrintJSON(definition)
      : definition
    )
  }
  ...
}
```

Reply:
I tried that style but reverted as it causes an error when using node.js 10:
https://github.com/SmartBear/swaggerhub-cli/pull/133/checks?check_run_id=1092640964